src/Frontend/Path.hs

```diff
@@ -14,10 +14,11 @@ module Frontend.Path
     , UserPs(..)
     , AdminPs(..)
     , IdeaMode(..)
+    , CommentMode(..)
     , viewIdea, editIdea, commentIdea, createIdea, listIdeas, listTopicIdeas
     , likeIdea, voteIdea, judgeIdea
     , voteCommentIdea, voteCommentIdeaReply, voteCommentWithContext
-    , replyCommentIdea, commentOrReplyIdea, isPostOnly, isBroken
+    , replyCommentIdea, commentOrReplyIdea, isPostOnly, isBroken, onComment
```

Comment:
It seems that the `onComment` is not used outside of the module, also seems it is just the abstract common part of some others. It shouldn't be in the export list, or am I missing something?